zerver/lib/import_realm.py

```diff
@@ -1303,3 +1306,78 @@ def import_analytics_data(realm: Realm, import_dir: Path) -> None:
     re_map_foreign_keys(data, 'analytics_streamcount', 'stream', related_table="stream")
     update_model_ids(StreamCount, data, 'analytics_streamcount')
     bulk_import_model(data, StreamCount)
+
+    fix_datetime_fields(data, 'analytics_fillstate')
+    handle_fill_state(data, 'analytics_fillstate', realm)
+
+# To Handle FillState we have to compare the state of new (current) server to the old
+# (imported realm) server. If the new server is currently empty, we just import
+# FillState from the old server, if the old server is ahead, we delete any entries
+# from the imported realm's analytics tables and if the new server is ahead, then
+# we have to count stats just for the target Realm.
+def handle_fill_state(data: TableData, table: TableName, realm: Realm) -> None:
+    fill_state = FillState.objects.all()
+    if fill_state is None:
+        # Assuming server is empty.
+        # TODO: A better condition is required here.
+        update_model_ids(FillState, data, 'analytics_fillstate')
+        bulk_import_model(data, FillState)
```

Comment:
I would do an early return here, and then we don't need all the code below to be unnecessary indented in an `else` block.